services/ens/api.go

```diff
@@ -75,6 +83,18 @@ func (api *API) Stop() {
 	})
 }
 
+func (api *API) GetEnsUsernames(ctx context.Context) ([]*EnsUsernameDetails, error) {
```

Comment:
As you are adding new API, is it going to work for mobile?

Reply:
After reading the code, maybe this should not be public?

Reply:
Sorry by not public I mean, the function name being lowercase, that would make it private

Reply:
Nope indeed, if called from RPC they need to be capitalized